### DISCUSSION TITLE
Elastalert Query

### DISCUSSION THREAD

**Discussion by vikrampc** [jertel/elastalert2#770] [Q&A] [answered]
Hello,

I wanted to write a query for rule mentioned below. Request you to kindly help me on that.

Below is Rule Condition:
In wazuh SIEM if there is level of alert is 12 and above in agent 000 and if the same source IP has generated the alert level 12 and above in any other agent then alert should be triggered.

**Comment by jertel** [comment, marked as answer]
Please read the project documentation and example rules first, then attempt to build your rule on your own. When you see errors, or the rule doesn't trigger as expected, then search the documentation, FAQ, and these discussions forums for similar discussions, as someone has very likely already encountered a similar issue. Please see #11 for more guidance.
